charts/gxf/values.yaml

```diff
@@ -55,6 +55,9 @@ resources:
     # secretName: your-secret
     # secretKey: your-secret-key
 
+httpsConnector:
+  enabled: false
```

Comment:
Wil je deze niet op true zetten?

Reply:
Dit is een generieke GXF chart. Dit op true zetten zou een breaking change zijn voor andere gebruikers.